model/schema.sql

```diff
@@ -0,0 +1,18 @@
+-- add schema info here. Do not include when pushing to master.
+CREATE DATABASE hiHuman;
+USE hiHuman;
+
+CREATE TABLE "Profile" (
+  "id" INT( 11 ) 'AUTO_INCREMENT' NOT NULL,
+  "number" VARCHAR( 255) NOT NULL,
+  "email" VARCHAR(255) NOT NULL,
+
+  PRIMARY KEY ( `id` ) 
+);
+
+CREATE TABLE "Connections" (
+  "friendID" INT( 11 ) AUTO_INCREMENT NOT NULL,
+  "friendNumber" INT ( 255) NOT NULL,
+  "friendEmail" INT ( 255) NOT NULL,
+  PRIMARY KEY ("id") 
```

Comment:
This table doesn't need a primary key or the AUTO_INCREMENT for friendID.  It's just a lookup table.  You should model it with Sequelize, and set the association in the User model.

Reply:
Your model looks correct anyway, though